plugins/orchestrator-backend/src/service/SonataFlowService.test.ts

```diff
@@ -155,32 +148,23 @@ describe('SonataFlowService', () => {
     beforeEach(() => {
       jest.clearAllMocks();
     });
-    it('should return workflow execution response when the request is successful', async () => {
+    it('should propagate thrown error when the request is successful', async () => {
```

Comment:
if the request is successful, why we have an error to propagate?

Reply:
ithink it is by mistake. checking

Reply:
fixed